### DISCUSSION TITLE
Quick add note from a notification

### DISCUSSION THREAD

**Discussion by pbienst** [gsantner/markor#1662] [Ideas] [answered]
It would be nice if you could quickly add a note from a permanent notification. That way, you can add a note without leaving your current application, you just pull the notification down.

This is e.g. implemented in orgzly, so perhaps some code could be reused?

https://github.com/orgzly/orgzly-android

**Comment by harshad1** [comment]
There are a bunch of apps which allow one to add a shortcut to a notification. You could use one of these to add a shortcut to the 'share into' markor shortcut.

**Comment by pbienst** [comment]
Which one did you use? I tried notification bar launcher, but that one didn't work...

**Reply by deathblade666** [reply, marked as answer]
Just finding this and wanted to chime in. I use tasker for this purpose. Have a persistent notification that gets created on boot. With button to "create a note" and "create tasks". Clicking on create a note opens the quick note and create tasks opens the todo section of the app. Hope you find this helpful after so much time has pasted.
